### DISCUSSION TITLE
Power measuring for roller shutter

### DISCUSSION THREAD

**Discussion by kaese2002** [mongoose-os-apps/shelly-homekit#1476] [Ideas]
Hello

thx for the feature and finally supporting Eve.
But i noticed on a 2.5 it´s only working in switch mode. If you use a 2.5 in roller shutter or garage opener mode the feature isn´t present. Would be cool if you can fix that.

thx

**Comment by rojer** [comment]
@timoschilling i think it makes sense, can you add it?

**Comment by kaese2002** [comment]
Any news on this?

**Comment by timoschilling** [comment]
Need to find some time for that

**Comment by tteixeira** [comment]
+1

**Comment by bockph** [comment]
Hi

> 
> 

> thx for the feature and finally supporting Eve.
> But i noticed on a 2.5 it´s only working in switch mode. If you use a 2.5 in roller shutter or garage opener mode the feature isn´t present. Would be cool if you can fix that.


Hi can you describe how excactly Power measurement is integrated into eve? Or is the corresponding version not yet on the server for OTA update? I Have a 2.5 and 1PM and simply used the links from Readme. Toggling on off via homekit and eve app does work like a charm but no hint wor the PM values (via browser i can see them)

**Comment by timoschilling** [comment]
![IMG_A71F5655F0AD-1](https://user-images.githubusercontent.com/165599/133249210-5bb49167-3957-434e-ae64-0c93706c9cf8.jpeg)

**Comment by timoschilling** [comment]
Implementing it for garage opener would make sense, because garage opener only give a milliseconds impulse.
